build/build-image/cross/Dockerfile

```diff
@@ -71,7 +71,7 @@ RUN go get golang.org/x/tools/cmd/cover \
             golang.org/x/tools/cmd/goimports
 
 # Download and symlink etcd. We need this for our integration tests.
-RUN export ETCD_VERSION=v3.1.12; \
+RUN export ETCD_VERSION=v3.1.13; \
```

Comment:
Can we please move the upgrade to 3.1.13 to a separate PR?

Reply:
Will do.